### PR TITLE
added badges for code quality and coverage

### PR DESCRIPTION
**Pull request by junnbang**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -2,6 +2,8 @@
 ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.com/AY1920S1-CS2103T-F14-4/main[image:https://travis-ci.com/AY1920S1-CS2103T-F14-4/main.svg?branch=master[Build Status]]
+https://coveralls.io/github/AY1920S1-CS2103T-F14-4?branch=master[image:https://coveralls.io/repos/github/AY1920S1-CS2103T-F14-4/badge.svg?branch=master[Coverage Status]]
```
Comment (tysng) [inline README.adoc]: Didn't specify the `main` repo. See the netlify preview, this is not linked correctly

**Review by tysg — CHANGES_REQUESTED** [review]
Also update the `README` in the root folder if you free

File: README.adoc [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+image:https://api.codacy.com/project/badge/Grade/7e4a8637349c4a028a87709e1d573861["Codacy code quality", link="https://www.codacy.com/manual/junnbang/main?utm_source=github.com&utm_medium=referral&utm_content=AY1920S1-CS2103T-F14-4/main&utm_campaign=Badge_Grade"]
```
Comment (tysng) [inline README.adoc]: Also this is your fork right?

**Review by tysg — DISMISSED** [review]
Need to change the badge url later

**Comment by junnbang** [comment]
Updated to coverage badge to master branch, may require merge to master to update

**Review by tysg — APPROVED** [review]
Just gonna try merging it. Should be ok.

**Pull request merged**